extension/js/common/core/attachment.ts

```diff
@@ -120,6 +120,8 @@ export class Attachment {
       return 'publicKey';
     } else if (this.name.match(/\.asc$/) && this.length < 100000 && !this.inline) {
       return 'encryptedMsg';
+    } else if (this.name.match(/ATT[0-9]{5}$/) || this.type === 'application/pgp-encrypted') {
```

Comment:
Did you mean `&&`? If Outlook sets the attachement type as 'application/pgp-encrypted', do we really need to test attachment file name? Can we "download api export" one of such messages to have it in our "decrypt" tests or does it have PII? Is it encrypted-only or signed too?

Reply:
Thanks! I should have used `&&`. 

>If Outlook sets the attachement type as 'application/pgp-encrypted', do we really need to test attachment file name?

I'm not sure yet, because the filename test is dedicated to the Microsoft outlook weird email. Maybe the test helps in cases such as a user is forwarding a PGP message in the attachment and flowcrypt extension will let it be in the attachment section rather than treating it as an encrypted message and rendering it?

>Can we "download api export" one of such messages to have it in our "decrypt" tests or does it have PII? Is it encrypted-only or signed too?

Yes, we can. I have decrypted it, it is only a sample encrypted message with an attachment.

I also noticed that the message is already rendering into the FlowCrypt Inbox as follows:
<img width="1178" alt="Screen Shot 2021-12-17 at 3 16 57 PM" src="https://user-images.githubusercontent.com/46025304/146505640-e9af39c5-1947-479d-8e96-b476024a8110.png">

But I can't find the reference code used in Gmail for Web so the changes don't take effect in the Gmail Web app.
<img width="1173" alt="Screen Shot 2021-12-17 at 12 52 12 PM" src="https://user-images.githubusercontent.com/46025304/146505788-b659b23d-a12f-4b08-b86d-482d0706552e.png">